orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/DeregisterInstancesFromLoadBalancerTaskSpec.groovy

```diff
@@ -43,44 +45,55 @@ class DeregisterInstancesFromLoadBalancerTaskSpec extends Specification {
 
   def "creates a deregister instances from load balancer task based on job parameters"() {
```

Comment:
these test names would read slightly better the task name was set apart, maybe by single quotes